Librarian/Series.js

```diff
@@ -219,18 +215,17 @@ Series.prototype.run = (done)=> {
     return done();
   })
   .fail((err)=> {
-    log.info(err);
-    return done();
+    return done(err);
   });
 };
 
 
-Series.prototype.cron = ()=> {
+Series.prototype.cron = function cron() {
   return this._defer(this.run.bind(this));
 };
 
 
-module.exports = (opts)=>  {
-  var _opts = opts || {};
-  return new Series(_opts);
+module.exports = (opts = {})=> {
```

Comment:
__[jshint]__
*  'default parameters' is only available in ES6 (use esnext option).

*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).